configuration/concepts/districts.csv

```diff
@@ -0,0 +1,174 @@
+uuid,Members,Answers,Fully specified name:pt_BR,Fully specified name:en,Short name:pt_BR,Short name:en,Data class,Data type,_version:1,_order:100
+0c648651-dfd2-4375-a878-4a442c6b7cb9,,,Alto Molocue,Alto Molocue,Alto Molocue,Alto Molocue,Misc,N/A,,
```

Comment:
@krishnanspace why are we using concepts here for districts, provinces and health facilities? From our last discussion I remember agreeing the use the address hierarchy module for these address details.

This approach is not easily maintainable and there isn't a way to create a link between these fields.

Reply:
But these are observation forms and the address hierarchy feature is not available in Observation forms. I remember last time agreeing to use it as concepts for Obs forms

Reply:
Ok so this approach is not ideal because we lose the references between these fields. We need to think about how we can incorporate the AH module here in an Obs form. Perhaps for this release we can push this through, but I want us to look into changing this to use AH like everywhere else in the system

Reply:
We need to be consisten, and having to maintain more than one source of address's will give us more work to do or introduce data inconsistencies.